### DISCUSSION TITLE
Manage organization by nested scope

### DISCUSSION THREAD

**Discussion by Mizugorou** [npm/rfcs#345] [Question]
Hi,
Is npm possible to manage organization by nested scope?
For example ,  `@organization/department`/package-name.
What I want is the subdirectory of organization that I can manage `department` in the same `organization`.
How do you think?

**Comment by ljharb** [comment]
`@organization/department` is already a potential package. How would clients know if `@organization/department/f` is a package, or an `f` entry point in the `@organization/department` package, without asking the server?
